README.md

```diff
@@ -105,7 +105,20 @@ vars:
     stripe__subscription_metadata: ['the', 'list', 'of', 'property', 'fields'] # Note: this is case-SENSITIVE and must match the casing of the property as it appears in the JSON
     stripe__customer_metadata: ['the', 'list', 'of', 'property', 'fields'] # Note: this is case-SENSITIVE and must match the casing of the property as it appears in the JSON
 ```
+**Note** 
 
+Alternatively, if you happen to be using a reserved word as a field in your metadata, similarly incompatible name, or just wish to rename your field, we have updated our variables to be able to accept a dictionary in addition to strings. Below is an example using `stripe__plan_metadata` of how you would add the respective fields to your root `dbt_project.yml` file.
+
+```yml
+vars: 
+  stripe__plan_metadata:
+    - metadata_field_1
+    - metadata_field_2
+    - name: incompatible.field
+      alias: rename_incompatible_field
+    - name: field_is_reserved_word
+      alias: field_is_reserved_word_xyz
+```
```

Comment:
I think this should replace the documentation for the metadata variables that you see above. I feel it is a bit duplicative. We could still include the note about the reserved word, but this way it is condensed into one section.

Similar to how we document the variables [here](https://github.com/fivetran/dbt_netsuite_source#passing-through-additional-fields). We should commit to instructing the customer to either write the variable. While we can support the ability to write the old way, we should instruct people to only write the variable in the new way. With the `name` or the `name` and the `alias`

Reply:
Just updated to condense with the prior info we had for metadata variables.